File: Arbitration/MPArbitration/Model/CaseWorkflowParams.cs
namespace MPArbitration.Model
{
    public enum CaseWorkflowAction
    {
        AssignCustomer = 3,
        AssignUser = 2,
        MarkRead = 0,
        MarkUnread = 1,
        NSARequestSentToPayor = 4,
        NotificationFailed = 5
    }

    public class CaseWorkflowParams
    {
        public int assignToId { get; set; }
        public int caseId { get; set; }
        public int customerId { get; set; }
        public CaseWorkflowAction action { get; set; }
        public string? message { get; set; }
        public string? JSON { get; set; } = null;
    }
}

File: Arbitration/MPArbitration/ClientApp/src/app/model/case-workflow-params.ts
export enum CaseWorkflowAction {
    AssignCustomer = 3,
    AssignUser = 2,
    MarkRead = 0,
    MarkUnread = 1,
    NSARequestSentToPayor = 4
}

export class CaseWorkflowParams {
    public caseId: number = 0;
    public customerId:number = 0;
    public action:CaseWorkflowAction | undefined;
    public assignToId: number = 0;
}